src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,17 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String engin;
+    private float bucketcapacity;
```

Comment:
pls don't forget to use camelCase for variables to have better readability
```suggestion
    private float bucketCapacity;
```